examples/demo/updating_plot/updating_plot2.py

```diff
@@ -36,15 +36,19 @@
 
 
 class AnimatedPlot(HasTraits):
-    def __init__(self, x, y, color="blue", bgcolor="white"):
-        self.x_values = x[:]
-        self.y_values = y[:]
+    x_values = Array()
+    y_values = Array()
+    color = Str()
```

Comment:
This is because I noticed it was never being set any time the class was instantiated.  ie we would do something like `AnimatedPlot(x, jn(i, x), color)` and `bgcolor` would always just be he default.
If it is not used in this example code, and is not configurable in the UI I figured it could just be fixed